webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/components/DirectiveEditForm.scala

```diff
@@ -207,6 +206,20 @@ class DirectiveEditForm(
         DisplayColumn.FromConfig
       ).display
     }
+    SHtml.ajaxButton(
+      "Target rules",
```

Comment:
it needs a loader indicator, else when there is a lot of rules, we come in a blank page and nothing tell us that something will happen.